NuRadioReco/modules/trigger/rnog_surface_trigger.py

```diff
@@ -0,0 +1,214 @@
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.modules.trigger.highLowThreshold import get_majority_logic
+from NuRadioReco.framework.trigger import RNOGSurfaceTrigger
+from NuRadioReco.utilities import units
+import NuRadioReco.utilities.fft
+import numpy as np
+import scipy.signal
+import copy
+import time
+import logging
+
+logger = logging.getLogger('rnog_surface_trigger')
+
+def schottky_diode(trace, threshold, temp=250*units.kelvin, Vbias=2*units.volt):
+    '''
+    Returns the absolute maximum of the diode response depending on the maximum input signal.
+    Options for temperature are [300K, 273K, 250K], for the bias voltages [2, 1.5, 1, 0.5]V.
+    The triggerpath reduces the voltage by -10dB, hence the threshold is small. The linear fit is obtained from
+    measurements done at DESY Zeuthen.
```

Comment:
docstring nitpicking - can you add an empty line after the first line here? Docstrings should start off
```
short single line description

(optional) longer description
```

Reply:
Thanks, I was not aware of that.